.github/workflows/dailies.yml

```diff
@@ -36,62 +30,6 @@ jobs:
       python-versions: ${{ needs.load_python_and_os_versions.outputs.ALL_PYTHON_VERSIONS }}
       os-versions: ${{ needs.load_python_and_os_versions.outputs.ALL_OS_VERSIONS }}
 
-  run-daily-dev-tests:
-    needs: load_python_and_os_versions
-    uses: ./.github/workflows/dev-testing.yml
-    secrets:
-      DANDI_API_KEY: ${{ secrets.DANDI_API_KEY }}
-      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      S3_GIN_BUCKET: ${{ secrets.S3_GIN_BUCKET }}
-    with:
-      python-versions: ${{ needs.load_python_and_os_versions.outputs.ALL_PYTHON_VERSIONS }}
-
-  run-daily-live-service-testing:
-    needs: load_python_and_os_versions
-    uses: ./.github/workflows/live-service-testing.yml
-    secrets:
-      DANDI_API_KEY: ${{ secrets.DANDI_API_KEY }}
-      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      S3_GIN_BUCKET: ${{ secrets.S3_GIN_BUCKET }}
-    with:
-      python-versions: ${{ needs.load_python_and_os_versions.outputs.ALL_PYTHON_VERSIONS }}
-      os-versions: ${{ needs.load_python_and_os_versions.outputs.ALL_OS_VERSIONS }}
-
-  run-daily-neuroconv-docker-testing:
-    uses: ./.github/workflows/neuroconv_docker_testing.yml
-    secrets:
-      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-      S3_GIN_BUCKET: ${{ secrets.S3_GIN_BUCKET }}
```

Comment:
This one should stay in the main dailies workflow since it just tests the latest release of neuroconv with docker.

Reply:
I reverted but I does not look like it tests a release version.